control-plane/tests/test_cli_inspection.py

```
@@ -1696,6 +1696,194 @@ def post_json(path: str, payload: dict[str, object]) -> dict[str, object]:
                     servers[0].shutdown()
                 thread.join(timeout=2)
 
+    def test_long_running_runtime_surface_exposes_cited_advisory_review_routes(self) -> None:
+        store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                host="127.0.0.1",
+                port=0,
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)